### DISCUSSION TITLE
Creation and editing art -images of skycultures

### DISCUSSION THREAD

**Discussion by PLANETARAS** [Stellarium/stellarium#2192] [General]
What do the numbers represent after the name of the png image (And andromeda.png 198 215 3881 337 136 3092 224 428 9640) in the file "constellationsart in the skycultures folder? Is it possible to change the parameters of representation in the sky or add records for new images? Es. Superimposing the "Big Dipper" on the Ursa Major. Can anyone have suggestions help me? Thank you all!

**Comment by gzotti** [comment]
Please read the User Guide, chapter 9.

**Comment by PLANETARAS** [comment]
Thank you very much very useful Inviato da Posta per Windows Da: Georg ZottiInviato: mercoledì 19 gennaio 2022 23:44A: Stellarium/stellariumCc: PLANETARAS; AuthorOggetto: Re: [Stellarium/stellarium] Creation and editing art -images of skycultures (Discussion #2192) Please read the User Guide, chapter 9.—Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you authored the thread.Message ID: ***@***.***>
